source/wp-content/themes/wporg-showcase-2022/inc/block-config.php

```diff
@@ -70,12 +70,19 @@ function ( $a, $b ) {
 		_n( 'Popular tags <span>%s</span>', 'Popular tags <span>%s</span>', $count, 'wporg' ),
 		$count
 	);
+	$option_labels = array_map(
+		function( $name, $count ) {
+			return $name . " ($count)";
+		},
+		wp_list_pluck( $tags, 'name' ),
+		wp_list_pluck( $tags, 'count' )
+	);
```

Comment:
How about adding a local utility function to reuse for all these?

Reply:
Added in 468d05315ea3096cd25e30c1e9006403fc0169d2